### ISSUE TITLE
Electron-rebuild succeeded but `require("zeromq");` fails at runtime in Electron App

### ISSUE THREAD

**Issue by mikelyndersOKCC** [zeromq/zeromq.js#409]
**Describe the bug**

Hello, I am trying to use `zeromq` in the render process of an electron application. My understanding is that I need to rebuild it for electron. I think I went through the proper steps (see below) but I still get the following error in the electron dev tools console:

`Error: No native build was found for platform=browser arch=javascript runtime=electron abi=undefined uv= libc=glibc node=undefined webpack=true`

Any ideas?

**Reproducing**

*in electron app*
- `npm install electron-rebuild`
- `npm install zeromq`
- open git-bash (mingw64) shell (I'm on windows, followed advice from here #396)
- `$(npm bin)/electron-rebuild`
- *side note: had some serious issues getting the right Visual Studio build tools. Apparently 2019 will not work. I had to uninstall 2019 and use `npm install windows-build-tools` to get 2017 which eventually worked

**Expected behavior**
I expected no errors in the console and the line `const zmq = require("zeromq");` to succeed in the render process of an electron app.

**Tested on**
 - OS: Windows 10
 - ZeroMQ.js version: [6.0.0-beta.6]

**Comment by shirik534** [comment]
@mikelyndersOKCC, I have the same problem. Did you manage to solve it?

**Comment by mikelyndersOKCC** [comment]
@shirik534 No. I ended up going a different direction with my application and Zero MQ became no longer necessary.
